testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -166,12 +166,14 @@ class SettingsMenuActivity : BaseActivity() {
         launch {
             try {
                 MiniApp.instance(AppSettings.instance.miniAppSettings).listMiniApp()
+                URL("https://www.example.com$urlParameters").toURI()
```

Comment:
If I remember correctly, we automatically add the `?` to the params string. So I think this should be:
```suggestion
                URL("https://www.example.com?$urlParameters").toURI()
```

Reply:
updated.